src/pages/hire-developers/hiring-hassle.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import StopWatch from "../../../static/stopwatch.svg";
+import Retention from "../../../static/retention-01.svg";
+import FastTime from "../../../static/fast-time.svg";
+import Cost from "../../../static/cost-01.svg";
+import Hours from "../../../static/dev-match-time-1.svg";
+import Container from "../../components/UiComponent/Container";
+
+import { HiringHassleWrapper } from "./style";
+
+const HiringHassle = () => {
```

Comment:
add this json constant file and import here

Reply:
done